test/test_torch.py

```diff
@@ -2115,12 +2115,11 @@ def test_inference(default_dtype):
 
     def test_qengnie(self):
         qengines = torch._C._supported_qengines()
-        # [TODO] Enable after the interface change
-        # original_qe = torch._C._get_qengine()
-        # for qe in qengines:
-        #     torch._C._set_qengine(qe)
-        #     assert torch._C._get_qengine() == qe, 'qengine not set successfully'
-        # torch._C._set_qengine(original_qe)
+        original_qe = torch._C._get_qengine()
```

Comment:
actually, looks like user is going to use `toch.backends.quantized.engine = 'qnnpack'` to set the engine? if so we should do this here instead.